### PR TITLE
docs: fix typo buildModules to modules for nuxt3

### PR DESCRIPTION
**Pull request by sathishvm**
Need to change the prop name buildModules to modules for nuxt3 example

### CODE REVIEW

File: packages/docs/ssr/nuxt.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ By default `@pinia/nuxt` exposes one single auto import: `usePinia()`, which is
 // nuxt.config.js
 export default {
```
Comment (posva) [inline packages/docs/ssr/nuxt.md]: can you add the defineNuxtConfig as above?

**Review by posva — CHANGES_REQUESTED** [review]
Thanks!

**Review by posva — CHANGES_REQUESTED** [review]
Thanks!

File: packages/docs/ssr/nuxt.md (hunk 1 above)
Comment (sathishvm) [reply packages/docs/ssr/nuxt.md]: sure, will add it

**Pull request merged**